automation/repeated_test.sh

```diff
@@ -152,7 +169,10 @@ for lane in "${TEST_LANES[@]}"; do
     for i in $(seq 1 "$NUM_TESTS"); do
         echo "test lane: $lane, run: $i"
         make cluster-sync
-        ginko_params="-no-color -succinct -slow-spec-threshold=30s -focus=${NEW_TESTS} -skip=QUARANTINE -regexScansFilePath=true"
+        ginko_params="-no-color -succinct -skip=QUARANTINE -randomize-all"
```

Comment:
`-randomize-all` don't we use the same seed for every run?

Reply:
I think the seed is set for usual test runs from outside in [automation/test.sh](https://github.com/kubevirt/kubevirt/blob/3f40002e29c9938d8e7b28e70124d1dffce02caf/automation/test.sh#L335). We don't use it here.